### PR TITLE
Development

### PR DESCRIPTION
**Pull request by Klimpavlov**

### CODE REVIEW

File: PartyPlaylist/HomePage/spotifyClone/SearchPage.css [hunk 1]
```diff
@@ -161,3 +161,74 @@ a:hover {
...
+.popularity {
+    color: white;
+    padding-top: 5px;
+    margin-left: -25px;
+}
+
+.section-content {
+    display: flex;
+    justify-content: space-between;
+    padding-left: 700px;
```
Comment (AlesBan) [inline PartyPlaylist/HomePage/spotifyClone/SearchPage.css:184]: I think it's not good idea
try to do 2 columns

File: PartyPlaylist/HomePage/spotifyClone/SearchPage.css [hunk 2]
```diff
@@ -161,3 +161,74 @@ a:hover {
...
+    margin: 0px 10px;
+    font-size: 10px;
+    text-align: center;
+    font-weight: 600;
+    background-color: ghostwhite;
+    color: black;
+    box-shadow: 0 8px 15px rgba(0, 0, 0, .3);
+    transition: .3s;
+}
+.floating-button:hover {
```
Comment (AlesBan) [inline PartyPlaylist/HomePage/spotifyClone/SearchPage.css:230]: cool👍
but make pls transform bit smaller

File: PartyPlaylist/HomePage/spotifyClone/SearchPage.html [hunk 3]
```diff
@@ -68,5 +68,96 @@
...
+        <div class="section-titles">
+            <h2 class="section-title">Треки</h2>
+            <h4 class="popularity">Популярность</h4>
+            <h2></h2>
+            <h2></h2>
+        </div>
+        <div class="section-content">
+            <div class="track">
+                <div class="trackInfo">
+                    <a href="#">
```
Comment (AlesBan) [inline PartyPlaylist/HomePage/spotifyClone/SearchPage.html:84]: this links are useless

File: PartyPlaylist/HomePage/spotifyClone/SearchPage.css [hunk 4]
```diff
@@ -161,3 +161,74 @@ a:hover {
...
+    color: white;
+}
+
+.floating-button {
+    text-decoration: none;
+    width: 65px;
+    height: 30px;
+    border-radius: 20px;
+    border-color: black;
+    margin: 0px 10px;
```
Comment (AlesBan) [inline PartyPlaylist/HomePage/spotifyClone/SearchPage.css:221]: make pls margin-left bigger

**Review by AlesBan — APPROVED** [review]

File: PartyPlaylist/HomePage/spotifyClone/SearchPage.css (hunk 4 above)
Comment (AlesBan) [reply PartyPlaylist/HomePage/spotifyClone/SearchPage.css:221]: margin-right*

Comment (AlesBan) [reply PartyPlaylist/HomePage/spotifyClone/SearchPage.css:221]: the same with the account block on the top left

Comment (Klimpavlov) [reply PartyPlaylist/HomePage/spotifyClone/SearchPage.css:221]: > the same with the account block on the top left

what is the account block?

File: PartyPlaylist/HomePage/spotifyClone/SearchPage.html (hunk 3 above)
Comment (Klimpavlov) [reply PartyPlaylist/HomePage/spotifyClone/SearchPage.html:84]: actually no, except <a href>, i can explain it later

Comment (Klimpavlov) [reply PartyPlaylist/HomePage/spotifyClone/SearchPage.html:84]: actually no, i can explain it later

Comment (AlesBan) [reply PartyPlaylist/HomePage/spotifyClone/SearchPage.html:84]: 👌

File: PartyPlaylist/HomePage/spotifyClone/SearchPage.css (hunk 4 above)
Comment (AlesBan) [reply PartyPlaylist/HomePage/spotifyClone/SearchPage.css:221]: this
         <div class="dropdown">
            <button class="dropdown-button" id="open-search-form">
                <span class="user-icon">
                    <i class="user"></i>
                </span>
                <span class="text-bold">
                    username
                </span>
                <span>
                    <i class="chevron-down"></i>
                </span>
            </button>
            <ul class="dropdown-content" id="open-dropdown">
                <li>Account</li>
                <li>Logout</li>
            </ul>
        </div>

Comment (AlesBan) [reply PartyPlaylist/HomePage/spotifyClone/SearchPage.css:221]: wow, how it changed script into view

**Pull request merged**